src/core/include/units/bits/prime.h

```diff
@@ -73,7 +73,7 @@ template<std::size_t N, typename Callable>
 constexpr void call_for_coprimes_up_to(std::size_t n, const std::array<std::size_t, N>& basis, Callable&& call)
 {
   for (std::size_t i = 0u; i < n; ++i) {
-    if (std::apply([&i](auto... primes) { return ((i % primes != 0) && ...); }, basis)) {
+    if (std::apply([&i](auto... primes) noexcept { return ((i % primes != 0) && ...); }, basis)) {
```

Comment:
@JohelEGP, what is your opinion on explicitly marking lambdas as `noexcept`. I fear it will spread to almost all of the lambdas in our library.

Reply:
I personally never do that, even if I would add `noexcept` to the equivalent function.

Reply:
Yes, this is also experience.

Reply:
FYI: That always triggers `-Wnoexcept`, because of conditional `noexcept` statements in stdlibc++s tuple code.